lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -40,6 +47,44 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     }
   }
 
+  function handleKeyDown(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (event.isDefaultPrevented()) {
+      return;
+    }
+
+    const tabElements = event.currentTarget.parentElement?.children;
+
+    const getNextElementFromEventKey = (key: string) => {
+      switch (key) {
+        case 'Left':
```

Comment:
From MDN:
> [1] Internet Explorer, Edge (16 and earlier), and Firefox (36 and earlier) use "Left", "Right", "Up", and "Down" instead of "ArrowLeft", "ArrowRight", "ArrowUp", and "ArrowDown".

We don't support any of those browser versions, so I don't think these aliases are needed here